app/api/routes/helpers.py

```diff
@@ -0,0 +1,32 @@
+from prometheus_client import Counter, Summary
+
+from app import utils as utils
+from app.models import Category, Language
+
+logger = utils.setup_logger('routes_logger')
+latency_summary = Summary('request_latency_seconds', 'Length of request')
+failures_counter = Counter('my_failures', 'Number of exceptions raised')
+
+
+def _unauthorized_response():
```

Comment:
Now that we have a separate file for `helpers`, I don't think this needs to start with an underscore anymore. Either that, or `get_attributes` should be `_get_attributes`. I'd like these to be uniform, I don't care which we go with.

Reply:
I checked and the only time we have a `_` in front of functions is for the ones that start with two (like `__eq__`) and the time that we `import db as _db`. I don't remember why we don't just import it as `db`, but I remember there was a reason. Something to do with having `db` in the same scope at some point, I'm not sure.

Either way, this project doesn't really use the convention of `_function_name` for helper functions, so I'm going to rename this as `unauthorized_response()`.

If someone wants to go in and create a convention of doing that for helper functions, they're welcome to, but they need to do it for all of them rather than a random one-off helper function.